Example/Podfile

```diff
@@ -4,3 +4,7 @@ use_frameworks!
 target 'ReusableDemo' do
   pod 'Reusable', :path => '../'
 end
+
+post_install do |installer|
+  system "sed -i '' -E 's/EMBEDDED_CONTENT_CONTAINS_SWIFT[[:space:]]=[[:space:]]YES/EMBEDDED_CONTENT_CONTAINS_SWIFT = NO/g' Pods/Target\\ Support\\ Files/Pods-ReusableDemo/*.xcconfig"
```

Comment:
No, I copied this from a CocoaPods issue thread and changed the target. I'll switch to this style so it doesn't take a UNIX expert to figure out what that line is doing.